source/MaterialXGenShader/ColorManagementSystem.cpp

```diff
@@ -88,4 +88,107 @@ ShaderNodePtr ColorManagementSystem::createNode(const ShaderGraph* parent, const
     return shaderNode;
 }
 
+
+void ColorManagementSystem::getPortConnections(ShaderGraph* graph, ShaderNode* colorTransformNode,
+                                               const TypeDesc* targetType, GenContext& context,
+                                               ShaderInput*& inputToConnect, ShaderOutput*& outputToConnect)
+{
+    const std::string colorTransformNodeName = colorTransformNode->getName();
+    ShaderOutput* colorTransformNodeOutput = colorTransformNode->getOutput(0);
```

Comment:
Why not just do:
outputToConnect = colorTransformNode->getOutput(0);
inputToConnect = colorTransformNode->getInput(0);
?

Reply:
I used these two local variables later on when doing connections.